doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1831,9 +1831,10 @@ unspecified dummy Ltac1 closure and should not be further used.
 Switching between Ltac languages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We recommend using the :opt:`Default Proof Mode` option to switch between tactic
-languages with a proof-based granularity. This allows to incrementally port
-the proof scripts.
+We recommend using the :opt:`Default Proof Mode` option or the :cmd:`Proof Mode`
+command to switch between tactic languages.  The option has proof-level
+granularity while the command has :term:`sentence`-level granulatiry. This
+allows incrementally porting proof scripts.
```

Comment:
It's good that this piece of advice distinguishes the command and option by their granularity, but what is written is wrong:
- `Proof Mode` has a proof-level granularity (it has an effect for the current proof and not beyond)
- `Default Proof Mode` has the standard scope of options (https://coq.inria.fr/refman/language/core/basic.html#locality-attributes-supported-by-set-and-unset), i.e., by default it has the scope of the default module, but it can also have the scope of the current section if `local` is used, or a larger scope if `export` or `global` is used.

Reply:
I think it's correct except for a typo, which I fixed.

> `Proof Mode` has a proof-level granularity (it has an effect for the current proof and not beyond)

`Proof Mode` has proof-level _scope_.  Its _granularity_ is sentence level because it takes place immediately, i.e. you can change the setting after any sentence, whereas `Default Proof Mode` is only applied at the beginning of each proof; changing the setting within a proof has no immediate effect.  And note that this not the full description of either of these constructs.

Agreed?

Reply:
Thanks for clarifying, I had not noted this difference, but you are right.